File: php_docstrings copy/latest.php
#!/usr/local/bin/php
<?php
/**
 * script latest.php
 * ---------------
 *
 * Retrieves the latest stable version of OPNsense by parsing the changelog file.
 *
 * *Params*
 * 
 * - changelogfile : str
 *     index: 1
 *     Path to the changelog file (default: /usr/local/opnsense/changelog/index.json)
 * 
 * *Arguments*
 * 
 * - None
 * 
 * *Returns*
 * 
 * - string
 *     The latest stable version of OPNsense, or the current version if unknown.
 */

File: autodocs/scripts/firmware/latest.py
"""
script latest.php
---------------------------------------------------------------------

Retrieves the latest stable version of OPNsense by parsing the changelog file.

*Params*- arsv[1] : str
    -   Path to the changelog file (default: /usr/local/opnsense/changelog/index.json)

*Returns* 
  -   string
    - The latest stable version of OPNsense, or the current version if unknown.
"""